### DISCUSSION TITLE
Welcome to VFormer Discussions!

### DISCUSSION THREAD

**Discussion by NeelayS** [SforAiDl/vformer#33] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by innat** [comment]
@NeelayS Hello
I was looking for vivit model and found this amazing repo and many other models. I was wondering if these translated models have come with pre-trained weight also? Thank you.

**Reply by abhi-glitchhg** [reply]
Hey there @innat ,

> I was wondering if these translated models have come with pre-trained weight also? 

Unfortunately due to resource constrains (GPUs) we could not train any models, so pretrained weights are not available. :(
